docs_src/examples/Ch-ExamplesSendingAndConsumingBinary.md

```diff
@@ -185,3 +185,49 @@ data inside the EdgeX Reading's `BinaryValue` field.
 Using this approach, an Event can be sent containing data containing an
 arbitrary, flexible structure. Use cases could be a Reading containing
 multiple images, a variable length list of integer read-outs, etc.
+
+### Creating A CBOR Payload For Use With PUT Commands
+
+To create a CBOR payload that can, for example, be used with `PUT` commands,
+we first need to create a CBOR decoder (from a given set of CBOR bytes),
+along with a CBOR code handle. We` decode into an EdgeX event where, as
+far as the codec is concerned, the Go library recommended for parsing
+into an EdgeX event is https://github.com/ugorji/go
+
+What follows below is the `golang` code that accomplishes the steps above:
+
+``` golang
+package main
+
+import (
+  “io/ioutil”
+
+  “github.com/ugorji/go/codec”
+)
+
+func main() {
+    // Set up some CBOR data.
+	result := make(map[string]string)
+	result["EnableRandomization_Binary"] = "true"
+
+	// Create a CBOR decoder from the CBOR bytes, along with a CBOR code handle
+	var h codec.Handle = new(codec.CborHandle)
+	var input = make([]byte, 0)
+	var dec *codec.Encoder = codec.NewEncoderBytes(&input, h)
+
+	// Decode into an EdgeX Event
+	err := dec.Encode(result)
+	check(err)
+
+	// Fetch binary data and write to a file
+	ioutil.WriteFile("/Users/johnpoe/Desktop/Decoded_Binary", input, 0644)
+
+func check(e error) {
+    if e != nil {
+        panic(e)
+    }
+}
```

Comment:
For sake of brevity in this code sample can we remove this function which is only used once? I think we can ignore the error in the code which references this function and simply put a comment stating that it is ignored and what the error indicates

Reply:
I'm biased because I wrote the first version of this document--but my suggestion would be to just remove the definition of check() here and leave the calls to it in, similar to the other snippet above.  check() should be defined outside of main() at any rate, and I think leaving the call to check() in is a better way to communicate "you should check this error" than a comment.

Reply:
Good point, I agree

Reply:
In light of the helpful feedback, I agree that retaining the call to `check()` in this `doc` is more intention-revealing 👍